spring-boot-jib/checkstyle.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We have PMD already configured for the entire project, so unless this checkstyle is necessary for jib to work (e.g. jib will fail if we have any star imports), then let's remove it.

Reply:
Sure, its not necessary here. Will remove it

Reply:
Removed the file